setup.py

```diff
@@ -40,7 +40,7 @@ def run(self):
 
         try:
             os.environ['EPYTHON'] = 'python{}.{}'.format(sys.version_info.major, sys.version_info.minor)
-            subprocess.check_call(['py.test', '-v', testpath,
+            subprocess.check_call(['py.test', '-v', testpath, '-s',
```

Comment:
Oops. It slipped thru.

Made to allow the STDIN/STDOUT to go unmodified thru py.test until the end of the test. Is the default here around me, but should not had slipped on this unrelated PR.